dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -0,0 +1,14805 @@
+- apiVersion: resource.k8s.io/v1alpha3
+  kind: ResourcePool
+  metadata:
+    creationTimestamp: null
+    name: nvidia-00-dgxa100
+  spec:
+    devices:
+    - attributes:
+      - name: uuid
+        string: GPU-c2fe2ade-955a-421f-976e-3f1dd7254234
+      - name: product-name
+        string: Mock NVIDIA A100-SXM4-40GB
+      - name: brand
+        string: Nvidia
+      - name: architecture
+        string: Ampere
+      - name: cuda-compute-capability
+        version: 8.0.0
+      - name: driver-version
+        version: 550.54.15
+      - name: cuda-driver-version
+        version: 12.4.0
+      - bool: true
+        name: mig-capable
+      - bool: false
+        name: mig-enabled
+      name: gpu-0
+      sharedCapacityConsumed:
+      - capacity: 40Gi
+        name: gpu-0-memory
```

Comment:
I don't think this needs to be a capacity, but it does need to be an attribute so that users can select on it.

Reply:
This does need to have capacities for all of the memory blocks then though (which is probably more consistent anyway).